tests/checkpoint/test_checkpoint.py

```diff
@@ -3718,3 +3719,686 @@ def test_newstyle_checkpoint_instantiates_and_produces_a_validation_result_when_
 
     assert len(context.validations_store.list_keys()) == 1
     assert results["success"] == True
+
+
+def test_newstyle_checkpoint_instantiates_and_produces_a_printable_validation_result_with_batch_data(
```

Comment:
This replaces the previously removed `print(results)` assertions.

Reply:
very helpful note(s). Thank you :)